packages/doc/src/components/Navigation/Navigation.jsx

```diff
@@ -19,9 +19,9 @@ import MDXElements from '../MDXElements';
  */
 const getSorting = kind =>
   ({
-    order: (a, b) => (a.order > b.order ? 1 : -1),
-    alphabetic: (a, b) => (a > b ? -1 : 1),
-  }[kind ?? 'order']);
+    alphabeticDescending: (a, b) => (a.title > b.title ? -1 : 1),
+    alphabeticAscending: (a, b) => (a.title > b.title ? 1 : -1),
+  }[kind ?? 'alphabeticDescending']);
```

Comment:
We use `order` to sort some [documentation](https://github.com/Gympass/yoga/blob/master/packages/doc/content/guidelines/product-content/grammar-and-mechanics/index.mdx?plain=1#L5). It is also a parameter of our frontmatter schema.

How is the impact of this change on our [guidelines](https://gympass.github.io/yoga/guidelines/product-content)?

Reply:
Thanks for pointing this out, it was really some lack of attention to the order defined in Markdown (We didn't noticed this in the previous PR and the guidelines page is broken since then). I've fixed it and made the code infer which sorting function to use based on the MDX having an `order` value or not. What do you think?

![image](https://user-images.githubusercontent.com/28108272/140534812-c40aec5b-6fba-4e29-a756-78d789ef4408.png)